wp-content/themes/cambodia-ict-camp/inc/template-parts/content-list-posts.php

```diff
@@ -6,66 +6,69 @@
 $no_blog_image = '';
```

Comment:
What is it used for?

Reply:
That is the list for Blog page (index.php).

Reply:
My question is about $no_blog_image, what is it using for?

Reply:
The file is not used anymore as it is replaced with `content-grid-3-cols.php`